prover/protocol/dedicated/byte32cmp/multi_limb_cmp.go

```diff
@@ -100,9 +100,9 @@ func CmpMultiLimbs(comp *wizard.CompiledIOP, a, b LimbColumns) (isGreater, isEqu
 		numLimbs        = len(a.Limbs)
```

Comment:
We need more scrutiny for this package:

- The way this package works is by:

      1. breaking the inputs into limbs (typically 256 bits broken in 32 bytes)
      2. do the comparison limb by limb and output {-1, 0, 1} for each limb
      3. linearly combine the outputs using 2**<id of the limbs> to obtain the "syndrom"

But if there are more than 31 limbs, the syndrom will overflow and our technique does not work

Reply:
So there should at least be an assertion that fires up if something like this happen